frontend/src/components/NavBar.tsx

```diff
@@ -6,6 +6,7 @@ export default function NavBar(){
         <div className="navbar">
             <Link className={"nav-item"} to="/TestPage">TestPage</Link>
             <Link className={"nav-item"} to="/Login">Login</Link>
+            <Link className={"nav-item"} to="/MainPage">MainPage</Link>
```

Comment:
```suggestion
            <Link className={"nav-item"} to="/">MainPage</Link>
```
As before, make your MainPage the default.